backend/de.metas.handlingunits.base/src/main/java/de/metas/handlingunits/picking/job/repository/PickingJobRepository.java

```diff
@@ -118,4 +120,40 @@ public Optional<PickingJob> getDraftBySalesOrderId(
 				.firstIdOnlyOptional(PickingJobId::ofRepoIdOrNull)
 				.map(pickingJobId -> PickingJobLoaderAndSaver.forLoading(loadingSupportServices).loadById(pickingJobId));
 	}
+
+	public boolean hasReadyToReview(final ImmutableSet<PickingJobId> pickingJobIds)
+	{
+		if (pickingJobIds.isEmpty())
+		{
+			return false;
+		}
+
+		return queryReadyToReview(pickingJobIds).anyMatch();
+	}
+
+	public List<PickingJob> getByIsReadyToReview(
+			@NonNull final ImmutableSet<PickingJobId> pickingJobIds,
+			@NonNull final PickingJobLoaderSupportingServices loadingSupportServices)
+	{
+		if (pickingJobIds.isEmpty())
+		{
+			return ImmutableList.of();
+		}
+
+		final ImmutableSet<PickingJobId> pickingJobIdsEffective = queryReadyToReview(pickingJobIds).listIds(PickingJobId::ofRepoId);
+
+		return PickingJobLoaderAndSaver.forLoading(loadingSupportServices).loadByIds(pickingJobIdsEffective);
+	}
+
+	private IQuery<I_M_Picking_Job> queryReadyToReview(final ImmutableSet<PickingJobId> pickingJobIds)
+	{
+		Check.assumeNotEmpty(pickingJobIds, "pickingJobIds is not empty");
+
+		return queryBL.createQueryBuilder(I_M_Picking_Job.class)
```

Comment:
please also do `.addOnlyActiveRecordsFilter` so we as supporters have the option of deactivating records if ne want